<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IhmHorametre</class>
 <widget class="QDialog" name="IhmHorametre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IhmHorametre</string>
  </property>
  <widget class="QSlider" name="curseurVertical">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="affichage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeTech">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ativerTech">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Activer Technicien</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButtonTech">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plusButtonTech">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
